app/main.py

```diff
@@ -1,8 +1,41 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int, brand: str) \
+            -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: int,
+                 clean_power: int, average_rating: int,
+                 count_of_ratings: int) -> None:
```

Comment:
Code Style issue: avoid using backslashes for line continuation. It's better to use parentheses for line continuation. This will make your code cleaner and easier to read.